notebooks/community/model_garden/model_garden_tfvision_image_classification.ipynb

```diff
@@ -0,0 +1,771 @@
+{
```

Comment:
<p>Move the H1 heading as the first line in the links cell below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done</p>